jrnl/Entry.py

```diff
@@ -37,18 +39,30 @@ def title(self):
             self._parse_text()
         return self._title
 
+    @title.setter
```

Comment:
These setters I don't think are being used by your solution. Would it make sense to drop them?

Reply:
I think it's good to leave them in. It sends a nudge to future contributors that they should use entry's accessors rather than underscore-prefixed internals.